### DISCUSSION TITLE
Added: LDM notes for Sept. 24, Sept. 26, Oct. 10, and Oct. 15

### DISCUSSION THREAD

**Discussion by agocke** [dotnet/csharplang#1952] [General]
[C# Language Design Notes for Sept. 24](https://github.com/dotnet/csharplang/blob/master/meetings/2018/LDM-2018-09-24.md)

F#/C# combined LDM
1. What new C# features are going to require work to get F# support?
1.  How can the design of C# features play well with the broader .NET environment?

[C# Language Design Notes for Sept. 26](https://github.com/dotnet/csharplang/blob/master/meetings/2018/LDM-2018-09-26.md)

1. Warning waves

[C# Language Design Notes for Oct. 10](https://github.com/dotnet/csharplang/blob/master/meetings/2018/LDM-2018-10-10.md)

1. Open issues for pattern matching

[C# Language Design Notes for Oct. 15](https://github.com/dotnet/csharplang/blob/master/meetings/2018/LDM-2018-10-15.md)

1. [Function pointers](https://github.com/dotnet/csharplang/blob/master/proposals/function-pointers.md)
2. [Readonly struct members](https://github.com/dotnet/csharplang/issues/1710)
3. Syntax for async streams

**Comment by yaakov-h** [comment]
> Jared proposes Git SHA.

I thought LDM notes were supposed to be unattributed? 🙃

**Comment by bondsbw** [comment]
Regarding `funcptr`, unnamed/inferred lambdas were discussed in #1694.

**Comment by gafter** [comment]
> I thought LDM notes were supposed to be unattributed?

I think this is "Jared" in the generic, non-specific sense. 🙃
